standard.md

```diff
@@ -6,17 +6,17 @@
+**3. Clear Ownership** | Ownership of assets that the digital public good produces must be clearly defined and documented. For example, through copyright, trademark or other publicly available information.
+**4. Platform Independence** | When the digital public good has mandatory dependencies that create more restrictions than the original license, proving independence from the closed component(s) and/or indicating the existence of functional, open alternatives that can be used without significant changes to the core product is required.
+**5. Documentation** | Digital public goods require documentation of the source code, use cases, and/or functional requirements. For content collections, this should include all relevant/compatible apps, software, or hardware required to access the content collection, and instructions regarding how to use it. For software solutions, this should be technical documentation that would allow a technical person unfamiliar with the project to launch and run the software. For data projects, this should be documentation that describes all the fields in the set, and provides context on how the dataset was collected, and how it should be interpreted.
+**6. Mechanism for Extracting Data and Content** | Digital Public Goods with non personally identifiable information (PII) design for possibility of extracting or importing non-PII data and content from the system in a non-proprietary format.
+**7. Adherence to Privacy and Applicable Laws** | Digital Public Goods must be designed and developed to comply with applicable privacy laws.
+**8. Adherence to Standards & Best Practices** | Digital Public Goods must be designed and developed to align with relevant standards, best practices, and/or principles. For example,[the Principles for Digital Development](https://digitalprinciples.org/principles/).
+**9. Do No Harm By Design** | Digital Public Goods must be designed to anticipate, prevent, and do no harm by design.
```

Comment:
One of the reasons (9) was written as it was, extended from the idea that DPG products (particularly software) are have some lack of control in how they are used given the core concepts of software freedom embedded within Open Source licenses. As such, there was at least by some of the authors some intent that the projects who produce the DPGs would also take additional mitigation steps to discourse misuse/abuse on behalf of the project and its participants (arguably the extent of its locus of control) would do no harm -- as described below this can be both through specific software design aspects but also through community norms and policies.

Reply:
Yes, community norms and standards are considered when evaluating the DPG. However, they may not be a necessary requirement, since not all DPGs have large communities, or can implement these norms. In such a case, we expect that there are stronger guardrails by design. We try to evaluate the overall potential to do harm on a case-to-case basis, more than looking for specific compliance to all best practices.